honeybee_radiance/modifier/material/mirror.py

```diff
@@ -0,0 +1,332 @@
+"""Radiance Mirror Material.
+
+http://radsite.lbl.gov/radiance/refer/ray.html#Mirror
+"""
+from __future__ import division
+from .materialbase import Material
+import honeybee.typing as typing
+
+
+class Mirror(Material):
+    """Radiance mirror material.
+
+    Mirror is used for planar surfaces that produce virtual source reflections. This
+    material should be used sparingly, as it may cause the light source calculation to
+    blow up if it is applied to many small surfaces. This material is only supported for
+    flat surfaces such as polygons and rings. The arguments are simply the
+    RGB reflectance values, which should be between 0 and 1.
+
+    An optional string argument may be used like the illum type to specify a different
+    material to be used for shading non-source rays. If this alternate material is given
+    as 'void', then the mirror surface will be invisible. This is only appropriate if
+    the surface hides other (more detailed) geometry with the same overall reflectance. 
+
+    Args:
+        identifier: Text string for a unique Material ID. Must not contain spaces
+            or special characters. This will be used to identify the object across
+            a model and in the exported Radiance files.
+        r_reflectance: Reflectance for red. The value should be between 0 and 1
+            (Default: 1).
+        g_reflectance: Reflectance for green. The value should be between 0 and 1
+            (Default: 1).
+        b_reflectance: Reflectance for blue. The value should be between 0 and 1
+            (Default: 1).
+        modifier: Material modifier (Default: "void").
+        alternate_material: An optional material may be used like the illum type to
+            specify a different material to be used for shading non-source rays. If this
+            alternate material is given as "void", then the mirror surface will be
+            invisible. This is only appropriate if the surface hides other (more
+            detailed) geometry with the same overall reflectance (Default: None).
```

Comment:
It might be worth clarifying here that `"void"` is the same as `None`, which I think it is but I'm not sure. I'm also really unsure here if typical practice is to always put a material in here with the same rgb reflectance as the mirror? Or is it more common to leave this as None?

Reply:
As discussed it is not the same.